external_http_lb_bucket_backend/main.tf

```diff
@@ -0,0 +1,120 @@
+# [START cloudloadbalancing_global_ext_bucket_buckets]
+# Create Cloud Storage buckets
+resource "random_id" "bucket_prefix" {
+  byte_length = 8
+}
+
+resource "google_storage_bucket" "bucket_1" {
+  name                        = "${random_id.bucket_prefix.hex}-bucket-1"
+  location                    = "us-east1"
+  uniform_bucket_level_access = true
+  storage_class               = "STANDARD"
+  // delete bucket and contents on destroy.
+  force_destroy = true
+}
+
+resource "google_storage_bucket" "bucket_2" {
+  name                        = "${random_id.bucket_prefix.hex}-bucket-2"
+  location                    = "us-east1"
+  uniform_bucket_level_access = true
+  storage_class               = "STANDARD"
+  // delete bucket and contents on destroy.
+  force_destroy = true
+}
+# [END cloudloadbalancing_global_ext_bucket_buckets]
+
+# [START cloudloadbalancing_global_ext_bucket_files]
+# Upload files
+resource "null_resource" "upload_cat_image" {
```

Comment:
instead of a null_resource, we let the docs use gsutil for this step without terraform, or use the  `google_storage_bucket_object` resource instead

Reply:
Fixed, thanks!